src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -647,7 +649,7 @@ public function deleteAction(Request $request)
             }
         }
 
-        return $this->redirectToRoute('admin_categories_index');
+        return $this->redirectToRoute('admin_categories_index', ['categoryId' => $idParent]);
```

Comment:
Is this the desired behaviour? IMHO, it is preferable to stay in current index page when I delete a category, especially when  I delete one category (which is the aim of this action). wdyt?